### ISSUE TITLE
ColAndreas Centos 7 GLIBC_2.27 not found

### ISSUE THREAD

**Issue by Abyss-z94** [Pottus/ColAndreas#79]
Hi, it me again, i try to run ColAndreas on my Centos 7 server but got that message.

`Failed (/lib/libm.so.6: version `GLIBC_2.27' not found (required by plugins/ColAndreas_static.so))`

I just try `yum install libc.so.6` but it said already installed lasted version.

**Comment by AmyrAhmady** [comment]
Yes cause that static build is actually built on Ubuntu, so for a working version on CentOS 7, you need to build the plugin yourself
PS: or you can wait for me to build it..

**Comment by Abyss-z94** [comment]
I just pick Static version not static_ubuntu on release sector, is that just for ubuntu :<

**Comment by AmyrAhmady** [comment]
there's build instruction in here: https://github.com/Pottus/ColAndreas/wiki/Instructions#linux

**Comment by Abyss-z94** [comment]
i just follow build instruction but can't make it.

`[root@ATM-5501138-01 build]# cmake -DCMAKE_BUILD_TYPE=Release ..
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:108 (message):
  Could NOT find Bullet (missing: BULLET_DYNAMICS_LIBRARY
  BULLET_COLLISION_LIBRARY BULLET_MATH_LIBRARY BULLET_SOFTBODY_LIBRARY
  BULLET_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:315 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake/Modules/FindBullet.cmake:76 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:6 (find_package)


-- Configuring incomplete, errors occurred!
See also "/root/ColAndreas/build/CMakeFiles/CMakeOutput.log".
[root@ATM-5501138-01 build]#`

I just install bullet physic with yum install bullet but still error. What should i do now
